csharp/ql/examples/qlpack.yml

```diff
@@ -1,4 +1,6 @@
 name: codeql/csharp-examples
-version: 0.0.2
+groups: 
```

Comment:
Why has the version number been removed?

Reply:
We're not making any particular versioning guarantees for the example packs. They're just sample code that sits in a Git repo for customers to look at, rather than an officially supported pack that gets published to the package registry and consumed by real Code Scanning runs.